frontend/src/routes/+layout.server.ts

```diff
@@ -4,7 +4,8 @@ import type { LayoutServerLoadEvent } from './$types'
 import { USER_LOAD_KEY } from '$lib/user';
 import { getRootTraceparent } from '$lib/otel/otel.server'
 
-export async function load({ locals, depends, fetch }: LayoutServerLoadEvent) {
+export async function load({ locals, depends, fetch, request }: LayoutServerLoadEvent) {
+  const requestLang = request.headers.get('Accept-Language')?.split(',')[0]?.split(';')[0]?.split('-')[0]?.toLowerCase();
```

Comment:
added this as SSR would be wrong compared to what js determined the language should be, meaning the language would change after hydration.

Reply:
Nice. I refactored it, because there's a `getLocaleFromAcceptLanguageHeader()` available